pkg/minikube/driver/driver.go

```diff
@@ -279,7 +279,11 @@ func Suggest(options []registry.DriverState) (registry.DriverState, []registry.D
 				continue
 			}
 
-			ds.Rejection = fmt.Sprintf("%s is preferred", pick.Name)
+			if pick.Name == "" {
+				ds.Rejection = "Rejected due to low priority"
```

Comment:
Let's make this more specific and actionable. What do you think about this message:

`Not selected: "experimental" drivers must be specified using --driver`

Reply:
Needs some other word than "experimental" though, the `none` and `ssh` are more like opt-in.

`Not selected: "%s" driver must be specified explicitly using --driver` or something to that effect ?

Reply:
Updated. Thanks for your advice.

Used "Not selected: %s must be specified explicitly using --driver" in accordance with the the previous string
"%s is preferred"